static/style/inverted-style.css

```diff
@@ -360,18 +360,22 @@
   }
   .toolbar {
     position: fixed;
-    width: 100vw;
+    width: 100%;
     height: 48px;
     top: 0;
+    padding: 0 5px;
     backdrop-filter: blur(8px);
     display: flex;
     border-bottom: 1px solid #7f7f7f20;
     -webkit-backdrop-filter: blur(8px);
     align-items: center;
+    -webkit-box-sizing: border-box;
+    -moz-box-sizing: border-box;
+    box-sizing: border-box;
   }
   #authentication {
     position: absolute;
-    right: 0;
+    right: 5px;
```

Comment:
Doesn't seem right should be a multiple of 4px

Reply:
there is a 1px border